src/NUnitFramework/mock-assembly/mock-assembly.csproj

```diff
@@ -4,11 +4,20 @@
     <TargetFrameworks>$(NUnitRuntimeFrameworks)</TargetFrameworks>
     <RootNamespace>NUnit.Tests</RootNamespace>
     <IsTestProject>false</IsTestProject>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
What is the reason for adding this? 
Is it so that the XML analyzer run as I don't see much use in creating XML docs for an nunit test.

Reply:
Yes, without that one we get a warning like the one below
![image](https://github.com/nunit/nunit/assets/203432/18e8f2a2-bbe7-4f91-a64d-06ae1925cee4)

There is a roslyn issue on this, and an explanation for the behaviour - which affects the IDE0005 (using statement analysis)  :  https://github.com/dotnet/roslyn/issues/41640#issuecomment-588403027